From 352e18b766650d298a3ba54f757259220746b03b Mon Sep 17 00:00:00 2001
From: Diego Biurrun <diego@biurrun.de>
Date: Tue, 6 Nov 2012 12:07:35 +0100
Subject: [PATCH 187/278] x86: avresample: Add missing colons to assembly
 labels

YASM accepts labels without colons, but NASM issues warnings.
---
 libavresample/x86/audio_convert.asm |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/libavresample/x86/audio_convert.asm b/libavresample/x86/audio_convert.asm
index e7f7157..567a916 100644
--- a/libavresample/x86/audio_convert.asm
+++ b/libavresample/x86/audio_convert.asm
@@ -247,7 +247,7 @@ cglobal conv_s16p_to_s16_2ch, 3,4,5, dst, src0, len, src1
     add       src1q, lenq
     lea        dstq, [dstq+2*lenq]
     neg        lenq
-.loop
+.loop:
     mova         m0, [src0q+lenq       ]
     mova         m1, [src1q+lenq       ]
     mova         m2, [src0q+lenq+mmsize]
@@ -716,7 +716,7 @@ cglobal conv_fltp_to_flt_2ch, 3,4,5, dst, src0, len, src1
     add  src1q, lenq
     lea   dstq, [dstq+2*lenq]
     neg   lenq
-.loop
+.loop:
     mova    m0, [src0q+lenq       ]
     mova    m1, [src1q+lenq       ]
     mova    m2, [src0q+lenq+mmsize]
-- 
1.7.9.5

